benchmarks/overrides_benchmark/README.md

```diff
@@ -0,0 +1,19 @@
+# `__torch_function__` micro-benchmarks
+
+This benchmark suite provides a systemic way to measure the performance of `__torch_function__` overhead.
+
+## Getting started
+### Initial Setup
+Install `py-spy` by doing:
```

Comment:
add "note that there are more extensive docs on using py-spy in CONTRIBUTING.md"